referenceframe/primitives.go

```diff
@@ -72,16 +71,3 @@ func InterpolateInputs(from, to []Input, by float64) []Input {
 	}
 	return newVals
 }
-
-// GetFrameInputs looks through the inputMap and returns a slice of Inputs corresponding to the given frame.
-func GetFrameInputs(frame Frame, inputMap map[string][]Input) ([]Input, error) {
```

Comment:
@biotinker Just realized this function will error if we pass a `solverFrame` which although is valid input seems like another point in favor of "get rid of this and tie it to the frame its operating on"